src/common/components/repository-input/index.js

```diff
@@ -33,6 +34,14 @@ export class RepositoryInput extends Component {
     return message;
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
There is an intermediate step I am working on that will call on a Redux action: webhook creation. As soon as the webhook is successfully installed, it can redirect to the repo builds page.